samples/AdalAndroidTestApp/MainActivity.cs

```diff
@@ -133,6 +136,31 @@ protected override void OnActivityResult(int requestCode, Result resultCode, Int
                 this.accessTokenTextView.Text = "Cache cleared";
             });
         }
+
+        private async void conditionalAccessButton_Click(object sender, EventArgs e)
+        {
+            this.accessTokenTextView.Text = string.Empty;
+            TokenBroker tokenBroker = new TokenBroker();
+            tokenBroker.Sts = sts;
+            EditText email = FindViewById<EditText>(Resource.Id.email);
+            tokenBroker.Sts.ValidUserName = email.Text;
+            string value = null;
+            try
+            {
+                string policy = "{\"access_token\":{\"polids\":{\"essential\":true,\"values\":[\"5ce770ea-8690-4747-aa73-c5b3cd509cd4\"]}}}";
+                value = await tokenBroker.GetTokenInteractiveAsync(new PlatformParameters(this, true), null, policy);
```

Comment:
I've tested E2E for Android with Broker, and OBO with a .NET web service. More E2E tests are in progress